test/secret/secret_test.go

```diff
@@ -77,7 +77,7 @@ var _ = Describe("secret test", func() {
 			secretObj.Data.HecToken = testenv.EncodeBase64(modifiedHecToken)
 			secretObj.Data.Password = testenv.EncodeBase64(modifedKeyValue)
 			secretObj.Data.Pass4SymmKey = testenv.EncodeBase64(modifedKeyValue)
-			testenv.UpdateSecret(deployment, testenvInstance.GetName(), secretObj)
+			testenv.UpdateSecret(deployment, testenvInstance.GetName(), secretObj, false)
```

Comment:
Add a check if the operation was successful.
```
err := testenv.UpdateSecret(deployment, testenvInstance.GetName(), secretObj, false /*delete*/)
Expect(err).To(Succeed(), "Unable to updated secret Object")
``` 

This will fail the test if update is not successful.